addons/l10n_tn/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# -*- encoding: utf-8 -*-
+
+{
+    "name": "Tunisia - Accounting",
+    'icon': '/l10n_tn/static/description/icon.svg',
```

Comment:
Please don't add this huge image; it doesn't follow the template
See https://github.com/odoo/odoo/pull/108617